src/locale/locales/fr/messages.po

```diff
@@ -1334,7 +1338,11 @@ msgstr "Nouvelle publication"
 
 #: src/view/shell/desktop/LeftNav.tsx:258
 msgid "New Post"
-msgstr "Nouvelle publication"
+msgstr "Nouv. publication"
```

Comment:
Do you think this could be translated as "Nouvelle Poste?"

For example, other sites use the french word "Poster" for "Post"

Reply:
Indeed, this could be translated to `Nouveau post`! (Yeah, `post` is _masculine_ in French, whereas `publication` is feminine, so it would be `Nouveau` and not `Nouvelle` in that case.)

I didn't want to change `publication` to `post` in my pull request because I didn't want, as a first contribution, change all mentions of `publications` into `post` — but I think it would be a good call, as users are used to `post` to refer to it, both in writing and in speaking.

If you don't mind this change, I can make the move from `publication` to `post` in that pull request. Waiting for your green light to do so :)

Reply:
Yeah, that would be great! 

Again, want to thank you a lot for your help here. Please add yourself to the `"Last-Translator: \n"
"Language-Team: \n"` header in the `messages.po` file if you haven't done so already!